crates/tuono/src/commands/dev.rs

```diff
@@ -84,39 +91,58 @@ pub async fn watch(source_builder: SourceBuilder) -> Result<()> {
         let mut should_reload_ssr_bundle = false;
         let mut should_reload_rust_server = false;
         let mut should_refresh_axum_source = false;
+        let mut paths_to_refresh_types: Vec<PathBuf> = vec![];
+        let mut removed_files_from_types: Vec<PathBuf> = vec![];
```

Comment:
Better to use `Vec::new()` than the macro when making an empty vector.

```suggestion
        let mut paths_to_refresh_types: Vec<PathBuf> = Vec::new();
        let mut removed_files_from_types: Vec<PathBuf> = Vec::new();
```

Reply:
Can you elaborate this? Aren't they the same thing ([source](https://doc.rust-lang.org/src/alloc/macros.rs.html#42))?

Reply:
I think I just read it when I was learning rust - it doesn't really make a difference, but when there's no need to use a macro it's slightly better to use the function. This [reddit comment](https://www.reddit.com/r/rust/comments/fs1g5h/comment/flz0im5/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) sums it up quite well, but from the [rest of the discussion](https://www.reddit.com/r/rust/comments/fs1g5h/vec_or_vecnew/), it seems to be just preference. It's up to you :)

Reply:
No I don't think there is one, we would have to write one. I don't know if it's worth it, but i can investigate more if you like.

Reply:
I think investigating how we could extend our clippy config could worth it. But indeed it does not make sense for now at our size!